src/site/stages/build/drupal/health-care-region.js

```diff
@@ -24,6 +24,64 @@ function sortServices(sortItem) {
     .value();
 }
 
+// Creates the past-events listing pages
+function createPastEventListPages(page, drupalPagePath, files) {
+  const sidebar = page.facilitySidebar;
+  // Events listing page
+  const allEvents = page.pastEvents;
+
+  // store past & current events
+  const pastEventTeasers = {
+    entities: [],
+  };
+
+  // separate current events from past events;
+  _.forEach(allEvents.entities, value => {
+    const eventTeaser = value;
+    const startDate = eventTeaser.fieldDate.startDate;
+    const isPast = moment().diff(startDate, 'days');
+    if (isPast >= 1) {
+      pastEventTeasers.entities.push(eventTeaser);
+    }
+  });
+
+  // sort past events into reverse chronological order by start date
+  pastEventTeasers.entities = _.orderBy(
+    pastEventTeasers.entities,
+    ['fieldDate.startDate'],
+    ['desc'],
+  );
+
+  // Past Events listing page
+  const pastEventsEntityUrl = createEntityUrlObj(`${drupalPagePath}`);
+
+  const pastEventsObj = Object.assign(
+    { allEventTeasers: pastEventTeasers },
+    { eventTeasers: pastEventTeasers },
+    { fieldIntroText: page.fieldIntroText },
+    { facilitySidebar: sidebar },
+    { entityUrl: pastEventsEntityUrl },
+    { title: page.title },
+    { alert: page.alert },
+  );
```

Comment:
Do we need `Object.assign` for this for some reason, like it creates copies of the arrays instead of having the properties share the same reference?

```suggestion
  const pastEventsObj = {
    allEventTeasers: pastEventTeasers,
    eventTeasers: pastEventTeasers,
    fieldIntroText: page.fieldIntroText,
    facilitySidebar: sidebar,
    entityUrl: pastEventsEntityUrl,
    title: page.title,
    alert: page.alert,
  };
```

Reply:
Good q, @bkjohnson , and honestly, I was just copying an existing pattern - I'll update per your recommendation. If I don't report back, it means it worked!